### PR TITLE
Added scripts for 2016-11-10-moving-forward-together email

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

File: bin/mail/2016-11-10-moving-forward-together/send-test-emails [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# 359937 teachers
+../send-to-mailing-list --limit 360000 2016-11-10-moving-forward-together ../mailing-common/test-params-us.csv
```
Comment (ashercodeorg) [inline bin/mail/2016-11-10-moving-forward-together/send-test-emails]: Hopefully we aren't sending 360K test emails.

File: bin/mail/2016-11-10-moving-forward-together/generate-teachers [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      kind_s: user && role_s: teacher
+    ) || (
+      kind_s: Petition && role_s: educator
+    ) ||
+    kind_s: BringToSchool2013 ||
+    kind_s: ClassSubmission ||
+    kind_s: DistrictPartnerSubmission ||
+    kind_s: PLP interest form ||
+    kind_s: Teacher interest form ||
+    kind_s: School interest form ||
```
Comment (ashercodeorg) [inline bin/mail/2016-11-10-moving-forward-together/generate-teachers]: Do these multi-words need surrounding quotations?

File: bin/mail/2016-11-10-moving-forward-together/send-test-emails (hunk 1 above)
Comment (aoby) [reply bin/mail/2016-11-10-moving-forward-together/send-test-emails]: Oops, good catch. Fixed

File: bin/mail/2016-11-10-moving-forward-together/generate-teachers (hunk 2 above)
Comment (aoby) [reply bin/mail/2016-11-10-moving-forward-together/generate-teachers]: Hmm. Actually those aren't form types (space or no space). I don't know what these are :( We didn't catch that earlier [here](https://github.com/code-dot-org/code-dot-org/blob/a7ae172ebb006f34241f75c784df4bf54364a84e/bin/mail/2016-11-03-hoc-global-site-refresh/generate-teachers#L54-L56). I'll just remove them from this query.

Comment (ashercodeorg) [reply bin/mail/2016-11-10-moving-forward-together/generate-teachers]: Still need removal?

**Comment by ashercodeorg** [comment]
LGTM.

**Pull request merged**